api/api.proto

```diff
@@ -2518,7 +2613,7 @@ message CloudMigrateStartRequest {
 }
 
 // Response to start a cloud migration
-message CloudMigrateStartResponse {
+message SdkCloudMigrateStartResponse {
```

Comment:
Structure like this should instead be:

```
message SdkCloudMigrateStartRespone {
    CloudMigrateStartResponse response = 1;
}
```
So that we do not break the old REST interface. And consequently break drivers (like Portworx) that use these APIs